frontend/conf/routes

```diff
@@ -91,6 +91,7 @@ GET            /patrons                               controllers.Info.patron
 GET            /supporter                             controllers.Info.supporter
 GET            /us/supporter                          controllers.Info.supporterUSA
 GET            /eu/supporter                          controllers.Info.supporterEurope
+GET            /intl/supporter                        controllers.Info.supporterInternational
```

Comment:
OK, `int` makes more sense then. I used `intl` because it was mentioned in the Trello ticket: https://trello.com/c/3gH0adzO/161-international-supporter-row-eu-supporter-landing-pages